proto/ha_set_config.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package dash.ha_set_config;
+
+import "types.proto";
+
+enum Owner {
+    OWNER_UNSPECIFIED = 0;
```

Comment:
should we allow unspecified?

Reply:
I added `*_UNSPECIFIED` only for messages from SDN controller only, thought we should a default value? 

If it's agreed as mandatory, will just remove.